src/components/shared/Card/Card.js

```diff
@@ -131,15 +132,18 @@ const Card = ({
             />
             <div className="Card__warning">
               {warning}
+              {isRestrictedText}
               {partiallyAvailable}
-              <>
-                {" "}
-                Use the{" "}
-                <button type="button" className="Card__warning-button" onClick={handleScroll}>
-                  Monthly/Annual switch
-                </button>{" "}
-                at the top of the page to see the most recent available data.
-              </>
+              {isRestrictedText || partiallyAvailable ? (
```

Comment:
adding the `isRestrictedText` field and using that instead of `warning` allows us to ensure we add this additional "switch" text when that is set, but not just when `warning` is set. This is important as `warning` can be any general warning, and is utilized by the Jails tab, for which the switch text does not apply.